Formula/arangodb.rb

```diff
@@ -16,14 +18,38 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
+  fails_with :gcc do
+    build 730
+    cause "Generates incorrect code"
+  end
+
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  # the ArangoStarter is in a separate github repository;
+  # it is used to easily start single server and clusters
+  # with a unified CLI
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb/archive/0.14.0.tar.gz"
+    sha256 "43f9c6ecd492541c4616df523513898ad1512bec7baa245852ee4c401b786f2c"
+  end
+
   def install
     ENV.cxx11
 
+    resource("starter").stage do
+      ENV.append "GOPATH", Dir.pwd + "/.gobuild"
+      system "make", "deps"
```

Comment:
Does this install additional packages?

Reply:
The package is normally built from two different repositories. One is "github.com/arangodb/arangodb" and the other one "github.com/arangodb-helper/arangodb".